command/agent/config/test-fixtures/config-embedded-type.hcl

```diff
@@ -0,0 +1,30 @@
+pid_file = "./pidfile"
+
+auto_auth {
+	method {
```

Comment:
Did the method parsing also get updated?

Reply:
Yep

Reply:
Ah oops I'll fix